src/patternfly/components/SearchInput/search-input.scss

```diff
@@ -113,3 +129,20 @@
   display: flex;
   align-items: center;
 }
+
+.pf-c-search-input__menu {
```

Comment:
Should we use `pf-c-menu` here?

Reply:
Definitely could be, though since the menu is beta, and may be for a while, using it here would make the search input beta until the menu is promoted. Since this is really just a generic box and doesn't need the stuff the menu provides (items, groups, etc), I just made this something in the search input.

Reply:
wfm